Makefile

```diff
@@ -734,7 +734,11 @@ OPTIMIZE    = -O0
 LTO_FLAGS   = $(OPTIMIZE)
 else
 OPTIMIZE    = -Os
+ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
```

Comment:
Any reason for this? For example IO code will take quite big performance hit ...
With LTO compiler is able to inline functions from other files and further optimize them (constant passed as parameters may be folded inside function). Also register allocation may be improved (not sure if GCC already does this), keeping internal registers from inside of function...